recipes/brass/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+INST_PATH="."
+
+git clone git://git.code.sf.net/p/pstreams/code c++/pstreams
```

Comment:
Not easily/standard... checkout the `pstreams` repo and look at the `INSTALL`. They claim that the easiest way is to not having an install "just copy the .h file somewhere in your project"... doh :___(